packages/foam-core/src/model/workspace.ts

```diff
@@ -465,3 +487,19 @@ export class FoamWorkspace implements IDisposable {
     return workspace;
   }
 }
+
+// TODO move these utility fns to appropriate places
```

Comment:
This is not really meant to be addressed here, because I don't want to make the PR too big.
But basically we need to consolidate these utility functions, and export them in `foam-core`